etc/completions/flux-core

```diff
@@ -0,0 +1,45 @@
+ #
+ # Copyright 2014 Lawrence Livermore National Security, LLC
```

Comment:
I'm running bash version 4.4.19(1)-release (x86_64-pc-linux-gnu). It may also be that the extended pattern matching is not on. Try running `shopt -s extglob` to enable the extended glob that's failing.